File: PRAKTIKUM_4/soal2.py
batas = int(input('masukan batas maksimal :'))

for i in range(batas) :
    if i%2!=0:
        print(i,end=" ")
print()
j=batas
for j in range(batas,0,-1):
    if j%2==0:
        print(j,end=" ")

File: PRAKTIKUM_4/soal2.c
#include <stdio.h>
int main(){
    int batas,i,j;
    printf("masukan batas maksimal : ");
    scanf("%d", &batas);

    for(i=1;i<batas;i++){
       if (i%2==0) {
        printf("");
    }else {
    printf("%d ", i);
    }           
    }printf("\n");
    for(j=batas;j>=2;j--){
        if (j%2==0){
            printf("%d ",j);
        }else {
            printf("");
        }
    }
    return 0;
}